pkgs/servers/dico/default.nix

```diff
@@ -12,8 +12,10 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = [ "format" ];
 
+  nativeBuildInputs = [ groff ];
```

Comment:
Please set strictDeps if possible.

Reply:
Is it good practice to always set `strictDeps = true` if possible? I just never used strictDeps before.

Reply:
If you fixed cross once it is highly recommended to set it to keep updates from introducing regressions. You can also set it for new package right away.